### PR TITLE
1364: Add release notes for v2.6.0

### PR DESCRIPTION
**Pull request by benedeki**
* Release notes for v2.6.0
* _How to run_ document
* KafkaErrorSenderPlugin built-in plugin description
* Small text improvements

### CODE REVIEW

File: _docs/2.0.0/usage/run.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+- [Table Of Content](#table-of-content)
+- [Requirements and deploy](#requirements-and-deploy)
+- [Running Standardization](#running-standardization)
+- [Running Conformance](#running-conformance)
+- [Helper scripts](#helper-scripts)
+<!-- tocstop -->
+
+## Requirements and deploy
+
+For description of requirements and depploy see the [README.md](https://github.com/AbsaOSS/enceladus/blob/master/README.md#how-to-run)
```
Comment (AdrianOlosutean) [inline _docs/2.0.0/usage/run.md:22]:
```suggestion
For description of requirements and deployment see the [README.md](https://github.com/AbsaOSS/enceladus/blob/master/README.md#how-to-run)
```

File: _docs/2.0.0/usage/run.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+in the the project root.
+
+## Running Standardization
+```
```
[the remark this answers is not in the document]
Comment (benedeki) [reply _docs/2.0.0/usage/run.md:26]: FYI I just copied the code from project's README.md 😉

File: _posts/2020-05-29-release-2.6.0.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Menas Improvements
+
+- [#1250]({{ site.github.issues_url }}/1250) "Schema load from URL" - URL validation added - supports localhost, domain names, IP v4 and v6.
+- [#1234]({{ site.github.issues_url }}/1234) AVSC typed schema can be exported as same other types.
+
+## Menas Fixes
+
+- [#1347]({{ site.github.issues_url }}/1347) Each time a schema gets updated by uploading/remote-loading a new schema-file, the attachment is filed under the correct (same) version.
+- [#1347]({{ site.github.issues_url }}/1347) MongoDb remapping script added to [1347-attachment-version-off-by-one-mapping-script.js](https://github.com/AbsaOSS/enceladus/blob/master/scripts/mongo/1347-attachment-version-off-by-one-mapping-script.js)
```
[the remark this answers is not in the document]
Comment (benedeki) [reply _posts/2020-05-29-release-2.6.0.md:28]: Yes, on purpose. Two things addressed by the ticket. Is there a better way how to format that?

File: _docs/2.0.0/usage/run.md [hunk 4]
```diff
@@ -133,8 +138,7 @@ The list of options for configuring Spark deployment mode in Yarn and resource s
 | --conf-spark-memory-fraction **value**                               |             | **Advanced**. Fraction of (heap space - 300MB) used for execution and storage (default=`0.6`). Sets `spark.memory.fraction` Spark configuration parameter. See the detailed description [here](http://spark.apache.org/docs/latest/configuration.html#memory-management) |
 
 
-For more information on these options see the official documentation on running Spark on Yarn: 
-[https://spark.apache.org/docs/latest/running-on-yarn.html](https://spark.apache.org/docs/latest/running-on-yarn.html)
+For more information on these options see the official documentation on [running Spark on Yarn] [spark-running-yarn]
```
Comment (Zejnilovic) [inline _docs/2.0.0/usage/run.md:141]:
```suggestion
For more information on these options see the official documentation on [running Spark on Yarn][spark-running-yarn]
```

Does it work with the space?

Comment (benedeki) [reply _docs/2.0.0/usage/run.md:141]: Surprisingly it does. I will still consider fixing it. Good catch. 🙇

**Pull request merged**